index.html

```diff
@@ -0,0 +1,145 @@
+            <div id="txt_img4" class="img-txt2"><img class="img_txt" src="assets/images/txt_img4.svg"
+                    alt="projects name">
+                <p class="p3">A smart assistant to make driving more safe, efficient,
+                    and fun by unlocking your most expensive computer: your car.</p>
+                <img class="tags" src="assets/images/Tags2.svg" alt="language Tags">
+                <br>
+                <input class="btns" type="button" value="See Project">
+            </div>
+            <div id="card_img4" class="card right_align"><img class="card_img" src="assets/images/img4.svg"
+                    alt="project card"></div>
+        </div>
+    </article>
+
+    <section class="last_page">
+        <div class="left_block">
+            <h2 class="about">About <br> Myself</h2>
+            <div class="last_div">
+                <p class="last_P">Hello I’m a software developer! I can help you build a product , feature or website
+                    Look
+                    through some of my work and experience! If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+            </div>
+            <p class="P2">LET'S CONNECT
+            </p>
+            <div class="soc_img1 soc_img2">
+                <img src="assets/images/social_media_icons.svg" alt="socialmedia icons">
+            </div>
+            <input class="btns resume_btn" type="button" value="Get My Resume">
+        </div>
+        <div class="right_block">
+            <img id="skillz" src="assets/images/Skill-List.png" alt="List of skills">
+        </div>
+    </section>
+    <footer class="footer">
+        <div class="foot-image">
+            <h2 class="contactme">
+                Contact me
+            </h2>
+            <div class="lastP_div">
+                <p class="contact-info">
+                    If you have an application you are interested in developing,
+                    a feature that you need built or a project that needs coding.
+                    I’d love to help with it
+                </p>
+            </div>
+            <form action="https://formspree.io/f/form" method="post" id="form" class="contact-form">
+                <ul class="form-item">
+                    <li class="username"><input class="contact_input one_line" type="text" placeholder="username"
+                            name="username" maxlength="30" required></li>
+                    <li class="email"><input class="contact_input one_line" type="email" placeholder="Enter email"
+                            name="email" required></li>
+                    <li class="username"><textarea class="message contact_input" placeholder="Write your message here"
+                            maxlength="500" required></textarea></li>
+                </ul>
+                <input class="get_intouch btn1 form-button btns" type="submit" value="Get It Touch">
```

Comment:
- [ ] [OPTIONAL] i would suggest that you add a `cursor: pointer;` style to all of your button to change the cursor type when users hover on the buttons, that way your buttons will be more elegant and user friendly

before           |  after
:-------------------------:|:-------------------------:
![G1](https://user-images.githubusercontent.com/89282221/186756877-f7657a5c-d2ee-4421-ac8d-b34d524495eb.gif)  |  ![G2](https://user-images.githubusercontent.com/89282221/186756884-438d4a24-ddd3-40bc-87aa-73ced259cc57.gif)

Reply:
Thanks for this suggestion